src/main/java/com/puppycrawl/tools/checkstyle/checks/design/VisibilityModifierCheck.java

```diff
@@ -154,6 +154,29 @@
  * &lt;module name=&quot;VisibilityModifier&quot;/&gt;
  * </pre>
  * <p>
+ * Example with default values:
+ * </p>
+ * <pre>
+ * public class MyClass
+ * {
+ * private int myPrivateField1;
+ * public long serialVersionUID = 123456789L;
+ * public static final int MY_CONSTANT = 42;
+ * int myPrivateField3; // Violation, must have a visibility modifier
+ * protected String myProtectedField; // Violation, protected visibility is not allowed
+ * public final int MY_CONSTANT2 = 42; // Violation, public immutable fields are not allowed
+ * public int NOT_CONSTANT = 42; // Violation, not static final, immutable, matching the pattern
+ * public final java.lang.String notes; // Violation, public immutable fields are not allowed
+ * public final Set&lt;String&gt; mySet1 = new HashSet&lt;&gt;(); // Violation, HashSet is mutable
+ * public final ImmutableSet&lt;String&gt; mySet2; // Violation, mutable
+ * public final ImmutableMap&lt;String, Object&gt; objects1; // violation, Object class is mutable
+ * &#64;com.annotation.CustomAnnotation
+ * String annotatedString; // violation
+ * &#64;CustomAnnotation String shortCustomAnnotated; // violation
```

Comment:
Add spacing between examples.

Reply:
@0xbakry If you are resolving action items, please do not mark them as resolved, just comment "Done", if you have made the particular changes. Reviewer will mark them as resolved or you can mark them as resolved when the reviewer comments "resolved". This is the workflow we follow.